utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -464,3 +464,15 @@ fun Throwable.collectMessages(): List<String> {
  * Recursively collect the messages of this [Throwable] and all its causes and join them to a single [String].
  */
 fun Throwable.collectMessagesAsString() = collectMessages().joinToString("\nCaused by: ")
+
+/**
+ * Retrieve query parameters of an [URI]. Multiple values of a single key are supported if they are split by a comma, or
```

Comment:
Nits:
- Commit message: "Add *a* function"
- Docs: "an URI" -> "this URL" (as it's an extension function)